illumina_indexes.yaml

```diff
@@ -0,0 +1,387 @@
+##yaml file describing indexes for each lib prepr
```

Comment:
It's fine to have this file here. However, I feel that it's a bit unintuitive. Perhaps store it in amsystem or a google doc?